Search/search.tex

```diff
@@ -1105,6 +1105,8 @@ \subsection{Chart Renderer}
 | chart count by username limit 10
 \end{Verbatim}
 
+The ``other'' data group is a combined calculation of all remaining data groups that will not fit in the specified set of data series.  In the above examples we have limited the data series to 10 using the \code{limit 10} argument to chart, this means that the chart module will first survey the data as it comes in to pick the best data set to include and then add all other data sets to the ``other'' bucket.  Chart calculates the value of the ``other'' data set using the exact same math as the rendered data set, often leading to the ``other'' data set dominating the output of a chart.
```

Comment:
"We have limited the data series to 10 using the \code{limit 10} argument to chart, this means..."
That's not really what limit 10 means. This is a more general explanation of what happens regardless of whether limit 10 was applied or not.

Reply:
updated the wording, PTAL